File: Programmers/시저 암호.java
class Solution {
    public String solution(String s, int n) {
        String result = "";
        for (int i = 0; i < s.length(); i++) {
            if (s.charAt(i) != ' ') {
                int tmp = s.charAt(i);
                for (int j = 0; j < n; j++) {
                    tmp++;
                    if (tmp > 90 && tmp < 97 || tmp > 122) tmp -= 26;
                }
                char tmp2 = (char)tmp;
                result += tmp2;
            }
            else result += " ";
        }
        return result;
    }
}

File: Programmers/시저 암호.py
def solution(s, n):
    result = ""
    for i in range(len(s)):
        if s[i] == " ":
            result += " "
        else:
            asci_value = ord(s[i])
            if asci_value >= 65 and asci_value <= 90:
                if asci_value+n > 90:
                    asci_value = ((asci_value+n)-90)+64
                else:
                    asci_value += n
            else:
                if asci_value+n > 122:
                    asci_value = ((asci_value+n)-122)+96
                else:
                    asci_value += n
            result += chr(asci_value)
    return result
